docs/dom-testing-library/api-events.mdx

```diff
@@ -121,6 +121,8 @@ programmatically (such as `timeStamp`).
 const myEvent = createEvent.click(node, { button: 2 })
 fireEvent(node, myEvent)
 // myEvent.timeStamp can be accessed just like any other properties from myEvent
+// note: The access to the events created by `createEvent` is based on the native event API,
```

Comment:
Maybe it would be best to add a reference to the native event API (https://developer.mozilla.org/en-US/docs/Web/API/Event).

Reply:
@MatanBobi I think that this citation is more for users of the testing library, I'm not sure that in the native API you would even need to change the timestamp in the first place

Reply:
Oh yeah, I agree with you.. But since not all testing library users will be 100% familiar with the event API, I suggested to add a link to it.